.gitignore

```diff
@@ -1,56 +1,3 @@
-.DS_Store
-node_modules/
-/dist
-build
-/docs
-
-# local env files
-.env.local
-.env.*.local
-.env.*
-!.env.example
-
-# Log files
-npm-debug.log*
-yarn-debug.log*
-yarn-error.log*
-
-# Editor directories and files
-.idea
-.vscode
-*.suo
-*.ntvs*
-*.njsproj
-*.sln
-*.sw?
-/.firebase/
-.idea/
-*.iml
-
-# Coverage directory used by tools like istanbul
-coverage
-coverage.json
-coverageEnv
-/contracts-flat/
-
-/cache/
-/artifacts/
-
-.env
```

Comment:
WDYT about keeping `.env` in here for safety of us and anyone who has previously forked this repo?

Reply:
Looks like @yoshiwarab has implemented this, resolving 🙏